pkg/katatestutils/README.md

```diff
@@ -0,0 +1,116 @@
+# Kata test utilities
+
+* [Test Constraints](#test-constraints)
+    * [Usage](#usage)
+    * [Examples](#examples)
+        * [Skip tests based on user](#skip-tests-based-on-user)
+        * [Skip tests based on distro](#skip-tests-based-on-distro)
+    * [Full details](#full-details)
+
+This package provides a small set of test utilities.
+
+## Test Constraints
+
+This package provides helper functions to allow tests to be skipped based on a
+set of user-specified constraints.
+
+### Usage
+
+Create a `TestConstraint` object using the `NewTestConstraint()` constructor.
+This takes a single boolean parameter which specifies if debug output should
+be generated.
+
+In each test that needs to be skipped, call the `NotValid()` method on the
+`TestConstraint` object, passing one more more constraints that you want to be
+valid. The `NotValid()` function will return `true` if any of the specified
```

Comment:
Lines 25-27 suggested rewrite: 
The `NotValid()` function returns `true` if any of the specified constraints are not available. This allows for a more natural way to code an arbitrarily complex test skip as shown in the following example: